### DISCUSSION TITLE
Use managed identity?

### DISCUSSION THREAD

**Discussion by kasperbolarsen** [pnp/provision-assist-m365#67] [Ideas]
I am running my companies provisioning tool, very similar to this in principle (using LogicApp+ AzureFunctions),  and I can really recommend switching to using managed identity.  Just a suggestion 😁

**Comment by alexc-MSFT** [comment]
@kasperbolarsen thanks for this suggestion, it uses a Managed Identity for the automation components but not the Logic Apps yet, I'll add this onto our roadmap as this is a great idea to do it for the logic apps too, thanks! 👍
